sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/models/ServerSentEventListener.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.generic.core.http.models;
+
+import com.generic.core.util.ClientLogger;
+
+import java.io.IOException;
+
+/**
+ * Interface to implement event stream listeners for handling {@link ServerSentEvent}.
+ */
+public interface ServerSentEventListener {
+    ClientLogger LOGGER = new ClientLogger(ServerSentEventListener.class);
```

Comment:
This doesn't leak out as public API, does it?

Reply:
It was. Removed it and made the default logging in methods to do nothing instead.